deepmedic/routines/training.py

```diff
@@ -187,7 +187,9 @@ def do_training(sessionTf,
                 run_input_checks,
 
                 # ------- loggers -------
-                tensorboard_loggers=None
+                tensorboard_loggers=None,
+                # ---- Normalization ----
+                norm_params=None
```

Comment:
I would kindly ask us to remove the defaults from here.
You want to be defining the defaults only in trainSessionParams.py. Defaults concentrated there in one place.
And the code to throw an error if for any reason your SessionParams.py forgets to return a value (bug)

```suggestion
                norm_params
```

Also, move it further up, to the "pre-processing " section